data/transactions/logic/eval.go

```diff
@@ -5522,8 +5551,8 @@ func opBlock(cx *EvalContext) error {
 	}
 }
 
-// PcDetails return PC and disassembled instructions at PC up to 2 opcodes back
-func (cx *EvalContext) PcDetails() (pc int, dis string) {
+// pcDetails return PC and disassembled instructions at PC up to 2 opcodes back
+func (cx *EvalContext) pcDetails() (pc int, dis string) {
```

Comment:
Can we use this opportunity to replace newlines in the string with semicolons

Reply:
yeah, nice idea, since they are legal TEAL now.